src/series.jl

```diff
@@ -511,10 +511,17 @@ end
 
 # split the group into 1 series per group, and set the label and idxfilter for each
 @recipe function f(groupby::GroupBy, args...)
+    lengthGroup = maximum(union(groupby.groupIds...))
     for (i,glab) in enumerate(groupby.groupLabels)
         @series begin
             label     --> string(glab)
             idxfilter --> groupby.groupIds[i]
+            for (key,val) in d
+                if key != :group && isa(val, AbstractArray) && size(val,1) >= lengthGroup
```

Comment:
I think I disagree here, it seems like it could cause more trouble than benefit.